app/src/main/java/ba/sum/fpmoz/mim/ui/adapters/TeacherAdapter.java

```diff
@@ -81,11 +78,11 @@ public TeacherViewHolder(@NonNull final View itemView) {
                 Intent i= new Intent(itemView.getContext(), TeacherAdminEditActivity.class);
                 i.putExtra("USER_ID", key);
                 itemView.getContext().startActivity(i);
-            } );
+} );
             itemView.setOnClickListener((v) -> clickListener.OnClickListener(v, getAdapterPosition()));
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.